app/mashup/Preload.scala

```diff
@@ -95,7 +106,11 @@ final class Preload(
       .mon(_.lobby.segment("currentGame"))
       .zip(
         lightUserApi
-          .preloadMany(tWinners.map(_.userId) ::: entries.flatMap(_.userIds).toList)
+          .preloadMany(
+            forumTopics.flatMap(_.posts).flatMap(_.post.userId) ::: entries
+              .flatMap(_.userIds)
+              .toList
```

Comment:
yep good idea preloading those

Reply:
i'm not 100% sure but i'll say :) in case that's sarcasm